packages/containers-shared/src/utils.ts

```diff
@@ -61,22 +61,15 @@ export const runDockerCmd = (
 	};
 };
 
-export const runDockerCmdWithOutput = async (
-	dockerPath: string,
-	args: string[]
-): Promise<string> => {
-	return new Promise((resolve, reject) => {
-		execFile(dockerPath, args, (error, stdout) => {
-			if (error) {
-				return reject(
-					new Error(
-						`Failed running docker command: ${error.message}. Command: ${dockerPath} ${args.join(" ")}`
-					)
-				);
-			}
-			return resolve(stdout.trim());
-		});
-	});
+export const runDockerCmdWithOutput = (dockerPath: string, args: string[]) => {
```

Comment:
Out of curiosity what what this changed from async to sync? (not mentioned in the PR description).

Does it deserves a comment explaining why it should not be async?